src/scripts.js

```diff
@@ -53,6 +62,27 @@ function loadWeeklyHydrationData(hydration) {
   userWeeklyHydration.innerText = `Water Consumed This Week: ${userWeekHydration}oz`
 }
 
+function loadDailySleepData(sleep) {
+  let userDailySleepHrs = sleep.calculateDailyHoursSlept("2019/09/22");
+  let userDailySleepQuality = sleep.calculateDailySleepQuality("2019/09/22")
+  userDailySleep.innerText = `Hours Slept Today: ${userDailySleepHrs}hrs.
+  Sleep Quality Today: ${userDailySleepQuality}`
+}
+
+function loadWeeklySleepData(sleep) {
+  let userWeeklySleepHrs = sleep.calculateWeeklyHoursSlept("2019/09/22");
+  let userWeeklySleepQuality = sleep.calculateWeeklySleepQuality("2019/09/22")
+  userWeeklySleep.innerText = `Hours Slept This Week: ${Math.floor(userWeeklySleepHrs)}hrs.
+  Sleep Quality This Week: ${Math.floor(userWeeklySleepQuality)}`
+}
+
+function loadAvgSleepData(sleep) {
+  let userAvgSleepHrs = sleep.calculateAvgDailySleep("2019/09/22");
+  let userAvgSleepQuality = sleep.calculateAvgDailySleepQuality("2019/09/22")
+  userAvgSleep.innerText = `Average Hours Slept Today: ${Math.floor(userAvgSleepHrs)}hrs.
+  Sleep Quality Today: ${Math.floor(userAvgSleepQuality)}`
```

Comment:
I think there's an opportunity for (later) refactoring to combine daily/weekly functions for multiple types of data, or combine all sleep/hydration/activity data? Although not sure if making those DRY-er would violate SRP?